src/wp-admin/admin.php

```diff
@@ -68,24 +71,30 @@
 		 * If there are 50 or fewer sites, run every time. Otherwise, throttle to reduce load:
 		 * attempt to do no more than threshold value, with some +/- allowed.
 		 */
-		if ( $c <= 50 || ( $c > 50 && mt_rand( 0, (int)( $c / 50 ) ) == 1 ) ) {
+		if ( $c <= 50 || ( $c > 50 && mt_rand( 0, (int) ( $c / 50 ) ) == 1 ) ) {
 			require_once( ABSPATH . WPINC . '/http.php' );
-			$response = wp_remote_get( admin_url( 'upgrade.php?step=1' ), array( 'timeout' => 120, 'httpversion' => '1.1' ) );
+			$response = wp_remote_get(
+				admin_url( 'upgrade.php?step=1' ), array(
+					'timeout'     => 120,
+					'httpversion' => '1.1',
+				)
+			);
```

Comment:
What rule are we following here? Should this be more like this:

```php
$response = wp_remote_get(
	admin_url( 'upgrade.php?step=1' ),
	array(
		'timeout'     => 120,
		'httpversion' => '1.1',
	)
);
```

Reply:
Yeah, that looks wrong. Can you report it to the [WPCS repo](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues)?

Reply:
I can, but I wasn't sure if that's a rule we have because it's all over the code base now. Or should I create one for discussion first?

Reply:
If it doesn't look right, it's probably a bug. ;)